src/AuthProvider.tsx

```diff
@@ -76,11 +73,31 @@ export interface AuthProviderProps extends UserManagerSettings {
     onSignoutPopup?: () => Promise<void> | void;
 
     /**
-     * Allow passing a custom UserManager implementation
+     * Allow passing a custom UserManager
+     */
+    userManager?: UserManager;
+
+    /**
+     * @deprecated Allow passing a custom UserManager implementation
      */
     implementation?: typeof UserManager | null;
 }
 
+export interface AuthProviderUserManagerProps extends Omit<AuthProviderPropsBase, "redirect_uri" | "client_id" | "authority"> {
```

Comment:
`export` needed?

Reply:
the docs tool was giving a warning when these were not exported - if you are fine with that I can get rid of the exports